kitsune/wiki/views.py

```diff
@@ -1527,7 +1528,10 @@ def get_fallback_locale(doc, request):
     #   * wiki fallbacks for that locale
 
     for locale in all_accepted_locales:
-        if locale in translated_locales:
+        if locale == doc.locale:
```

Comment:
I think its better to add the ``en-US`` in the translation list rather than escaping it from here.